portal-ui/src/screens/Console/Buckets/ViewBucket/ViewBucket.tsx

```diff
@@ -389,6 +392,21 @@ const ViewBucket = ({
     }
   }, [loadingPolicy, setErrorSnackMessage, bucketName]);
 
+  useEffect(() => {
+    if (loadingPolicy) {
```

Comment:
This loading state should be loadingUsers as it is the one required for loading this information
```suggestion
    if (loadingUsers) {
```